owncloudData/src/main/java/com/owncloud/android/data/ClientManager.kt

```diff
@@ -29,6 +29,7 @@ import com.owncloud.android.lib.common.SingleSessionManager
 import com.owncloud.android.lib.resources.users.services.UserService
 import com.owncloud.android.lib.resources.users.services.implementation.OCUserService
 
+@Suppress("NAME_SHADOWING")
```

Comment:
Which variable names are being shadowed?

Reply:
Not sure why i introduced that, removing it..